CourseGrab/ViewControllers/Search/SearchDetailTableViewController.swift

```diff
@@ -36,8 +31,14 @@ class SearchDetailTableViewController: UITableViewController {
     }
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if indexPath.row == 0 {
+            let cell = SearchDetailCardTableViewCell()
```

Comment:
we should be dequeueing cells here! make sure you:
1) Register your custom cell by doing `tableView.register(...)`
2) Dequeue the cell here by doing `tableView.dequeueReusableCell(...)`